tgui/packages/tgui/interfaces/ClothingBooth/ItemSwatch.tsx

```diff
@@ -0,0 +1,62 @@
+import { classes } from 'common/react';
+import { Box, Tooltip } from '../../components';
+import { SwatchBisectLeft } from './swatchForegroundShapes/bisect-left';
+import { SwatchBisectRight } from './swatchForegroundShapes/bisect-right';
+import { SwatchPolkadots } from './swatchForegroundShapes/polkadots';
+import { SwatchClub } from './swatchForegroundShapes/club';
+import { SwatchDiamond } from './swatchForegroundShapes/diamond';
+import { SwatchHeart } from './swatchForegroundShapes/heart';
+import { SwatchSpade } from './swatchForegroundShapes/spade';
```

Comment:
```suggestion
import { SwatchBisectLeft, SwatchBisectRight, SwatchPolkdadots, /* etc */ } from './swatchForegroundShapes';
```
Add an `index.ts` file to the folder with the icons in it, which just re-exports the contents of that folder.
```
export { SwatchBisectLeft } from './bisect-left';`
// etc.
```
Putting here as related: the icon component files are components, so should follow PascalCase naming convention still.